app/controllers/customers_controller.rb

```diff
@@ -242,7 +262,16 @@ def finalize_merge
   def create
     @is_admin = true            # needed for choosing correct method in 'new' tmpl
     @customer = Customer.new(params[:customer])
+    @customer.password = params[:password] unless params[:password].blank?
+    @customer.password = params[:password_confirmation] unless params[:password_confirmation].blank?
+    puts "created by admin set to true"
```

Comment:
Ditto puts should be deleted

Reply:
again I don't remember why the blank? part is there, may not be necessary. I'll check.